index.js

```diff
@@ -634,7 +657,7 @@ function httpViewCleanup(db) {
   });
 }
 
-var localViewCleanup = utils.sequentialize(mainQueue, function (db) {
+function localViewCleanup(db) {
```

Comment:
This is the only function where you don't swap relatively straight forward from a single queue -> multiple queues. (But instead 'sequentialize' at a lower level.) I don't know enough about the underlying code to be able to check what the consequences of that are (if any race conditions might pop up etc.), but if you're reasonably confident that's handled, than consider this a :+1: (all the other stuff I understand ok and LGTM).